src/transformers/generation_utils.py

```diff
@@ -327,8 +391,11 @@ def generate(
         num_beam_groups: Optional[int] = None,
         diversity_penalty: Optional[float] = None,
         prefix_allowed_tokens_fn: Optional[Callable[[int, torch.Tensor], List[int]]] = None,
-        **model_kwargs
-    ) -> torch.LongTensor:
+        output_attentions: Optional[bool] = False,
```

Comment:
we always set all inputs to None in each `forward` and `generate()` method. The reason is that the default value is always defined in `model.config.output_attentions` and `model.config.output_hidden_states`